README.md

```diff
@@ -51,6 +61,20 @@ Some of the current and planned features are:
   - Iceberg (TODO)
   - Hudi (TODO)
 - Preloading DDL from `~/.datafusion/.datafusionrc` for local database available on startup
+- "Catalog File" support
+  - Save table definitions *and* data
+  - Save parquet metadata from remote object stores
```

Comment:
I think it would be useful if we had an issue that provided more details on aspirations here

Reply:
Filed https://github.com/datafusion-contrib/datafusion-tui/issues/122